.changes/unreleased/Under the Hood-20230316-123305.yaml

```diff
@@ -0,0 +1,6 @@
+kind: Under the Hood
+body: Add CommandComplete event, and fire it upon completion of every command
```

Comment:
Nit: `CommandCompleted`

Reply:
I corrected the typo in a fixup commit. Changie entry now has commit sha https://github.com/dbt-labs/dbt-core/pull/7180/commits/9860e1130da9db2684c48b6ad4b9a8f363922998